.github/workflows/main.yml

```diff
@@ -0,0 +1,26 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  ci:
+    uses: silverstripe/gha-ci/.github/workflows/ci.yml@v1
+    with:
+      dynamic_matrix: false
+      extra_jobs: |
+        - php: 7.4
```

Comment:
Generally better off using the dynamic matrix because it will stay up to date over time e.g. will automatically add an 8.2 job when silverstripe/gha-ci adds support for it - is there a strong reason you have a non-dynamic matrix here?

Reply:
@emteknetnz the `pgsql` tests fail with the standard matrix. I wasn't really trying to fix anything that is already broken, I just wanted to get us up to date with PHP `^8`.

Maybe I could raise an Issue regarding PostgreSQL and moving to the standard matrix?

Reply:
I think you should retain the dynamic matrix for better standardization

The failure in https://github.com/silverstripe/silverstripe-populate/actions/runs/3146840126/jobs/5115740952 look like it's a quoting issue.  There is likely a 'raw' sql like statement that in the populate codebase that just needs some double quotes put around it

Reply:
@emteknetnz I've searched around for `DB::` and `SQL`, but I can't find any raw queries being made. This is potentially a bit of a deeper issue?

Reply:
@emteknetnz I've raised a follow up Issue:
https://github.com/silverstripe/silverstripe-populate/issues/48

Hopefully this is acceptable, as we're not currently testing PostgreSQL anyway.

Reply:
Yeah I think we'll have to go for this for now. I did a quick investigation into the postgres failure though I didn't see anything obvious. For some reason `Title` is being converted to `title`, it may be something in either the framework or postgresql modules rather than populate